src/platform/forms/save-in-progress/SaveInProgressIntro.jsx

```diff
@@ -6,13 +6,10 @@ import moment from 'moment';
 
 import LoadingIndicator from '@department-of-veterans-affairs/formation-react/LoadingIndicator';
 import { getNextPagePath } from 'platform/forms-system/src/js/routing';
+import getBenefitString from '../../forms-system/src/js/utilities/benefit-description';
```

Comment:
Could. I tend to prefer relative paths instead of babel aliases because they play nicer with the `go to definition` functionality of my editor (which doesn't understand the babel aliases :disappointed: ).

In this case, because we're already importing from `platform/forms-system` right above, I can go ahead and change it to stay consistent. :+1: